packages/waas/src/base.ts

```diff
@@ -1,48 +1,53 @@
 import {
-  Intent,
-  SignedIntent,
+  changeIntentTime,
   closeSession,
+  combineTransactionIntents,
+  feeOptions,
+  finishValidateSession,
   getSession,
-  openSession,
+  getTransactionReceipt,
+  GetTransactionReceiptArgs,
+  initiateAuth,
+  Intent,
   listSessions,
-  validateSession,
-  finishValidateSession,
-  sessionAuthProof,
-  signIntent,
-  signMessage,
-  feeOptions,
+  openSession,
   sendDelayedEncode,
+  SendDelayedEncodeArgs,
   sendERC1155,
+  SendERC1155Args,
   sendERC20,
+  SendERC20Args,
   sendERC721,
+  SendERC721Args,
   sendTransactions,
-  combineTransactionIntents,
-  SignMessageArgs,
   SendTransactionsArgs,
-  SendERC20Args,
-  SendERC721Args,
-  SendERC1155Args,
-  SendDelayedEncodeArgs,
-  GetTransactionReceiptArgs,
-  getTransactionReceipt,
-  changeIntentTime
+  sessionAuthProof,
+  SignedIntent,
+  signIntent,
+  signMessage,
+  SignMessageArgs,
+  validateSession
 } from './intents'
-import { LocalStore, Store, StoreObj } from './store'
-import { newSession, newSessionFromSessionId } from './session'
-import { OpenSessionResponse } from './intents/responses'
-import { SimpleNetwork, WithSimpleNetwork, toNetworkID } from './networks'
+import {LocalStore, Store, StoreObj} from './store'
+import {newSession, newSessionFromSessionId} from './session'
+import {OpenSessionResponse} from './intents/responses'
+import {SimpleNetwork, toNetworkID, WithSimpleNetwork} from './networks'
 import {
+  IdentityType,
   IntentDataFeeOptions,
   IntentDataFinishValidateSession,
   IntentDataGetSession,
   IntentDataGetTransactionReceipt,
+  IntentDataInitiateAuth,
   IntentDataOpenSession,
   IntentDataSendTransaction,
   IntentDataSignMessage,
   IntentDataValidateSession
 } from './clients/intent.gen'
-import { SubtleCryptoBackend, getDefaultSubtleCryptoBackend } from './subtle-crypto'
-import { SecureStoreBackend, getDefaultSecureStoreBackend } from './secure-store'
+import {getDefaultSubtleCryptoBackend, SubtleCryptoBackend} from './subtle-crypto'
+import {getDefaultSecureStoreBackend, SecureStoreBackend} from './secure-store'
+import {keccak256, toUtf8Bytes} from "ethers/lib/utils";
```

Comment:
lets run `pnpm format` :)

Reply:
thanks, I didn't know about that actually 😅